reloj_checador_team03/app/assets/stylesheets/employees.scss

```diff
@@ -0,0 +1,11 @@
+// Place all the styles related to the Employees controller here.
+// They will automatically be included in application.css.
+// You can use Sass (SCSS) here: https://sass-lang.com/
+
+.table-thead {
+    background-color: #7AC143;
+}
+
+.table-thead-th{
+    color: #fff;
+}
```

Comment:
As you can see here, GitHub helps us to notice the lack of EOF in this file.
Here is a blog that describes better why is it needed:  https://thoughtbot.com/blog/no-newline-at-end-of-file

Sharing the blog in order to save some time trying to describe it by my self XD